tests/common/factories/annotation.py

```diff
@@ -131,7 +131,7 @@ def make_id(self, create, extracted, **kwargs):
             return
 
         # Ids in the DB are in hex, but in the code they should be URL safe
-        self.id = URLSafeUUID().process_result_value(  # pylint:disable=attribute-defined-outside-init
+        self.id = URLSafeUUID().process_result_value(  # pylint:disable=attribute-defined-outside-init,invalid-name
```

Comment:
I added `id` as a valid name on the main code's config, not on the tests. It only appears once so not as common (which makes sense as we don' usually declare attributes in test code).